app/services/spree_shopify_importer/data_savers/orders/order_creator.rb

```diff
@@ -25,12 +25,17 @@ def save!
         private
 
         def create_spree_order
-          order = Spree::Order.new(user: user)
+          @user = data_feed['customer'] ? user : fake_shopify_customer
```

Comment:
May be it's better to use `.present?` to check boolean value instead of nil of another object for `true` or `false`
`data_feed['customer'].present? ? user : fake_shopify_customer`

Reply:
also may be it's better to extract this line from `create_spree_order` method
```
        def save!
         @user = find_user
          Spree::Order.transaction do
            @spree_order = create_spree_order
            assign_spree_order_to_data_feed
            create_spree_line_items
            create_spree_payments
            create_spree_shipments
            create_spree_taxes
            create_spree_promotions
            create_spree_addresses
            create_spree_refunds
          end
          @spree_order.update_columns(timestamps)
        end

        private

        def find_user
          data_feed['customer'].present? ? user : fake_shopify_customer
        end
```